node-backend/migrations/20200508054633-update-recognition-hi5-table.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
```

Comment:
Don't need new migration currently. You should make change in current migration only. And we already discussed this